dinky-admin/src/main/java/org/dinky/service/impl/MonitorServiceImpl.java

```diff
@@ -74,27 +77,25 @@ public List<MetricsVO> getData(Date startTime, Date endTime, List<String> models
             throw new DinkyException("Please provide at least one monitoring ID");
         }
         endTime = Opt.ofNullable(endTime).orElse(DateUtil.date());
-        Timestamp startTS = Timestamp.fromLocalDateTime(DateUtil.toLocalDateTime(startTime));
-        Timestamp endTS = Timestamp.fromLocalDateTime(DateUtil.toLocalDateTime(endTime));
-
         if (endTime.compareTo(startTime) < 1) {
             throw new DinkyException("The end date must be greater than the start date!");
         }
 
-        Function<PredicateBuilder, List<Predicate>> filter = p -> {
-            Predicate greaterOrEqual = p.greaterOrEqual(0, startTS);
-            Predicate lessOrEqual = p.lessOrEqual(0, endTS);
-            Predicate local =
-                    p.in(1, models.stream().map(BinaryString::fromString).collect(Collectors.toList()));
-            return CollUtil.newArrayList(local, greaterOrEqual, lessOrEqual);
-        };
-        List<MetricsVO> metricsVOList =
-                PaimonUtil.batchReadTable(PaimonTableConstant.DINKY_METRICS, MetricsVO.class, filter);
-        return metricsVOList.stream()
-                .filter(x -> x.getHeartTime().isAfter(startTS.toLocalDateTime()))
-                .filter(x -> x.getHeartTime().isBefore(endTS.toLocalDateTime()))
-                .peek(vo -> vo.setContent(JsonUtils.parseObject(vo.getContent().toString())))
-                .collect(Collectors.toList());
+        String condition = startTime.getTime() + " <= heart_time AND heart_time <= " + endTime.getTime();
+        List<MetricsVO> metricsVOList = new ArrayList<>();
+        try(ResultSet read = SqliteUtil.INSTANCE.read(PaimonTableConstant.DINKY_METRICS, condition)) {
```

Comment:
rename `PaimonTableConstant` to `MetricsTableConstants` ???

Reply:
it's coding, maybe remove it at the end.